src/lib/net.js

```diff
@@ -175,7 +175,7 @@ foam.CLASS({
           return this.resp.json();
```

Comment:
Ignore file: Feature required universal HTTPRequest. Will merge in #248 after it's reviewed.